lib/libzrepl/data_conn.c

```diff
@@ -549,6 +554,20 @@ uzfs_zvol_rebuild_dw_replica(void *arg)
 		goto exit;
 	}
 
+	/* One more snapshot has been transferred */
+	if (hdr.opcode == ZVOL_OPCODE_REBUILD_SNAP_DONE) {
+		rc = uzfs_zvol_handle_rebuild_snap_done(&hdr,
```

Comment:
lets have an assert that current offset is >= vol_size

Reply:
Done